photoprism/.env-dist

```diff
@@ -0,0 +1,108 @@
+## Hardware Video Transcoding:
+# FFmpeg encoder ("software", "intel", "nvidia", "apple", "raspberry")
+# PHOTOPRISM_FFMPEG_ENCODER=software
+# FFmpeg encoding bitrate limit in Mbit/s (default: 50)
+# PHOTOPRISM_FFMPEG_BITRATE=32
+# FFmpeg encoding resolution limit in pixel height (default: 4096)
+# PHOTOPRISM_FFMPEG_RESOLUTION=1920
+## Run as a non-root user after initialization (supported: 0, 33, 50-99, 500-600, and 900-1200):
+PHOTOPRISM_UID=1000
+PHOTOPRISM_GID=1000
+PHOTOPRISM_UMASK=0000
+# MariaDB or MySQL database user password - this should match MARIADB_PASSWORD
+PHOTOPRISM_DATABASE_PASSWORD=insecure
+# this should match PHOTOPRISM_DATABASE_PASSWORD
+MARIADB_PASSWORD=insecure
+MARIADB_ROOT_PASSWORD=insecure
+
+## There's probably very little reason to change any values below here.
+# authentication mode (public, password)
+PHOTOPRISM_AUTH_MODE=password
+# disables HTTPS/TLS even if the site URL starts with https://
+PHOTOPRISM_DISABLE_TLS=true
+# file size limit for originals in MB (increase for high-res video)
+PHOTOPRISM_ORIGINALS_LIMIT=5000
+# improves transfer speed and bandwidth utilization (none or gzip)
+PHOTOPRISM_HTTP_COMPRESSION=gzip
+# log level: trace, debug, info, warning, error, fatal, or panic
+PHOTOPRISM_LOG_LEVEL=info
+# do not modify originals directory (reduced functionality)
+PHOTOPRISM_READONLY=false
+# enables experimental features
+PHOTOPRISM_EXPERIMENTAL=false
+# disables updating storage permissions via chmod and chown on startup
+PHOTOPRISM_DISABLE_CHOWN=false
+# disables built-in WebDAV server
+PHOTOPRISM_DISABLE_WEBDAV=false
+# disables settings UI and API
+PHOTOPRISM_DISABLE_SETTINGS=false
+# disables all features depending on TensorFlow
+PHOTOPRISM_DISABLE_TENSORFLOW=false
+# disables face detection and recognition (requires TensorFlow)
+PHOTOPRISM_DISABLE_FACES=false
+# disables image classification (requires TensorFlow)
+PHOTOPRISM_DISABLE_CLASSIFICATION=false
+# disables vector graphics support
+PHOTOPRISM_DISABLE_VECTORS=false
+# disables indexing and conversion of RAW images
+PHOTOPRISM_DISABLE_RAW=false
+# enables applying user presets when converting RAW images (reduces performance)
+PHOTOPRISM_RAW_PRESETS=false
+# a higher value increases the quality and file size of JPEG images and thumbnails (25-100)
+PHOTOPRISM_JPEG_QUALITY=85
+# automatically flags photos as private that MAY be offensive (requires TensorFlow)
+PHOTOPRISM_DETECT_NSFW=false
+# allows uploads that MAY be offensive (no effect without TensorFlow)
+PHOTOPRISM_UPLOAD_NSFW=true
+# use MariaDB 10.5+ or MySQL 8+ instead of SQLite for improved performance
+PHOTOPRISM_DATABASE_DRIVER=mysql
+# MariaDB or MySQL database server (hostname:port)
+PHOTOPRISM_DATABASE_SERVER=mariadb:3306
+# MariaDB or MySQL database schema name
+PHOTOPRISM_DATABASE_NAME=photoprism
+# MariaDB or MySQL database user name
+PHOTOPRISM_DATABASE_USER=photoprism
+MARIADB_AUTO_UPGRADE=1
+MARIADB_INITDB_SKIP_TZINFO=1
+MARIADB_DATABASE=photoprism
+MARIADB_USER=photoprism
```

Comment:
I see no reason to expose any of these variables

Reply:
fixed